packages/fx-core/src/component/drivers/aad/constants.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
The path name of file uses "drivers" instead of "driver". 
The names under component use singular style. 
Only a suggestion.

Reply:
Changed folder name to `driver`. For renaming `driver` to `action`, since other code in fx-core (e.g. deploy actions) already uses `driver` in multiple places, I will driver discussions offline and have additional PR to rename the folder, class names, etc.